software/README.md

```diff
@@ -2,6 +2,36 @@
 
```

Comment:
add H1 sized "Software" (for consistency with other folders)

Reply:
A single "#" means H1 right ?, it is already there, I didn't understand the issue :sweat_smile:

Reply:
Oh okay, it was hidden since it wasn't the changed part. Sorry. It's correct.